lib/server/controllers/gRPC/GRPCRunController.js

```diff
@@ -41,14 +41,24 @@ class GRPCRunController {
 
     // eslint-disable-next-line require-jsdoc
     async Update(runPatchRequest) {
-        const { run, relations } = this.gRPCToRunAndRelations(runPatchRequest);
-        const { runNumber } = run;
-        delete run.runNumber;
+        try {
```

Comment:
Here you don't need a try in the end?

Side note for you: try to avoid having huge try, because then you may have silent bugs. Only try the lines you expect to fail, not everything. The only exception is for example in the controller if you have a global catch to convert any error to a 500 response (for example)

Reply:
I think what you meant by this is because the RunService does not throw anymore I can leave the try behind. So, that is what I did, if this is not what you meant, please let me know and I will change it.